Libraries/DirectMLX.h

```diff
@@ -2676,9 +2676,38 @@ namespace dml
         return output;
     }
 
-    // 
-    // TODO: TopK (DML_OPERATOR_TOP_K1)
-    // 
+    struct TopKOutputs
```

Comment:
How about Values/Indices for consistency with MaxPoolingOutputs? I see that TopK in DirectML.h used Index though (that's my fault 😢). I'm fine either way.

    struct MaxPoolingOutputs
    {
        Expression values;
        Expression indices; // Only valid if outputIndices = true is supplied to MaxPooling()
    };

Reply:
I was also going to ask if we may support INT64 in this op. Perhaps we'll want to add a parameter for the output index data type with default UINT32... but only if this is something that may change soon.